### DISCUSSION TITLE
[Docs] Query on augmentation practices: using of random brightness and contrast

### DISCUSSION THREAD

**Discussion by j-shiau** [open-mmlab/mmpose#2879] [General]
### 📚 The doc issue

I've been exploring the field of human pose estimation (HPE) and noticed that most papers, following methodologies like Simple Baseline and HRNet, do not explicitly mention the use of random brightness and contrast as part of their data augmentation strategies. This observation led me to some curiosity regarding your stance on this matter.

I also saw your SKPS implementation within MMpose, and came across the application of random brightness and contrast augmentations. Given this, I am interested in understanding **if there is any experimental data available comparing the use of random brightness and contrast against not using them, within the same framework and settings.** Additionally, if no such data is available, I would appreciate **any educational guess or insights on why not incorporating random brightness and contrast could be a beneficial approach**.

Please note that while this query may not strictly fall under 'documentation issues', I chose to reach out through this channel due to the broad nature of the question, which spans beyond a single HPE paper. MMpose being a widely used framework in the field, seemed like the most suitable platform for this inquiry.

Thank you for your time and the incredible work your team has put into MMpose. Your insights would be greatly appreciated and beneficial to many in the HPE community.

### Suggest a potential alternative/fix

Some experimental data or educational guess about the use of random brightness and contrast against not using them.

**Comment by Ben-Louis** [comment]
Hello, regarding your questions,
1. The primary function of MMPose is to facilitate convenient ablation study. You can add brightness and contrast augmentation to any baseline config and compare model accuracy. Please refer to  [`Albumentation`](https://github.com/open-mmlab/mmpose/blob/main/mmpose/datasets/transforms/common_transforms.py#L571-L615) for details on adding such augmentation.
2. One of the most challenging aspects of pose estimation in practice is occlusion, OOD poses, and low image quality. Poor lighting conditions are typically not a significant obstacle for pose estimators.
